projects/plugins/jetpack/changelog/fix-gallery-image-with-link

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: bugfix
+
+Gallery: ensure clicks will navigate to custom URL when image has rounded corners.
```

Comment:
Nit: I think this is technically a fix to the Carousel feature rather than the Gallery code. It may be worth rephrasing this to represent that more accurately.
```suggestion
Carousel: ensure that clicks on rounded images in galleries will load a custom URL when it is specified.
```

Reply:
Ok, I've accepted this version.